cluster-autoscaler/cloudprovider/azure/test/templates/cluster-template-prow-aks-aso-cluster-autoscaler.yaml

```diff
@@ -257,7 +257,7 @@ spec:
         cluster-autoscaler-enabled: "true"
         cluster-autoscaler-name: ${CLUSTER_NAME}
         max: "5"
-        min: "1"
+        min: "0"
```

Comment:
Maybe having separate template for scale from zero would be preferred in general, since CAS behavior often differs due to node templating w/ min count == 0